### ISSUE TITLE
Sometimes relationships aren't loading

### ISSUE THREAD

**Issue by oskarrough** [internet4000/explorer-discogs#25]
Sometimes on https://discogs.radio4000.com/artists/2408160 the `members:hasMany:artist` relationship doesn't render all items properly. The list-item is there but the name is empty. If you refresh it'll either work or be different missing items. To be investigated.

<img width="294" alt="screen shot 2018-04-26 at 17 21 51" src="https://user-images.githubusercontent.com/184567/39315182-5577a3a4-4976-11e8-922c-a31236863a5f.png">

**Comment by hugurp** [comment]
i think it is because our Proxy API is hitting the rate limit, so it returns what it has?

**Comment by oskarrough** [comment]
Yes that's probably it. We better implement error handling in the API so we don't have to guess?

**Comment by oskarrough** [comment]
Error handling is in.
